ONSdigital/ras-frontstage
# Remove eq version code

# What and why?

Now that no exercises are using EQ v2 anymore, we can start removing all mentions of the old platform.  This PR ignores the version of EQ that the collection exercise states (which should be v3 anyway, but it's in theory still possible for v2 ones to be made) and just creates a v3 payload and goes to v3 regardless.

# How to test?

- Create exercise (manually or via acceptance tests)
- Set exercise to v2 (via UI or database)
- Have it go live and enrol for it
- Launch it, it should take you to the v3 endpoint in mock-eq.  You can also double check here that the eqVersion for the exercise is v2 to make sure.

# Trello
